### ISSUE TITLE
Ability to specify column headers

### ISSUE THREAD

**Issue by rpbouman** [just-bi/odxl#10]
Would be nice to allow the caller to specify the column headers. 

Possible solutions:
- extend ODATA $columns syntax to allow specification of alias.
- add new parameter
- add new meaning to the already existent header parameter.

**Comment by rpbouman** [comment]
The existing header query parameter can now accept a JSON array of strings, which will then be used as labels, or a JSON array of objects, of which the label propery will be used as label. 

The label will be used as SQL column alias so all output handlers will benefit (or, depending on how you look at it, will be affected). This means that if you specify labels, you must ensure they are unique. If they are not, un

You can still specify header=false to suppress headers for those output handlers for which this applies (csv, xlsx)

**Comment by joel-myworld** [comment]
Is there any way we can specify a color for the header cell? like the label?

**Comment by rpbouman** [comment]
Hi Joel, currently there is no way to do that. But it can be looked into. 

The way the header feature is designed allows for arbitrary information to be passed into ODXL. ODXL can be extended to do something with the header info. The original idea was to allow stuff like cell formats to be passed like that; the way that works in excel on the OOXML level is via styles. So I imagine it would not be very hard to also write other style properties, like text and background color.

Basically, we need a good feature request for this, and if possible, als someone that could figure out how to make this work on the OOXML level. I can of course do that but don't have the bandwidth right now. If you can find the developer resources in your company, then we will gladly accept pull requests. (If you are planning to dedicate some developer effort, then lets connect and discuss some details)
